open_seq2seq/data/text2speech/speech_utils.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import numpy as np
+import librosa
+import librosa.filters
+import math
+
+
+def get_speech_features_from_file(filename, num_features, pad_to=8,
+                                  features_type='magnitude',
+                                  window_size=1024,
+                                  window_stride=256,
+                                  mag_power=2,
+                                  feature_normalize=False,
+                                  mean = 0,
+                                  std = 1):
+  """
+  :param filename: WAVE filename
```

Comment:
This documentation format is not going to be rendered correctly. Can you convert it to Args: ... Returrns: ... as in some other places (Google documentation style). Also the first line of the doc should briefly describe what the function is doing.